app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
Thanks for the tip about `simple_form` I haven't really used it before; however, it seems that there are only a few different elements that you can do that with but `title` isn't one of them. As I said, I am fairly unfamiliar with `simple_form` so if anyone knows otherwise please let me know.